File: VoidDrops_auto/parse_prime.py
from html.parser import HTMLParser
from urllib import request

def main():
    response = request.urlopen('https://forums.warframe.com/forum/3-pc-update-build-notes/')
    parser = MyHTMLParser()
    parser.feed(str(response.read()))


class MyHTMLParser(HTMLParser):
    #def handle_starttag(self, tag, attrs):
    #    if tag == "p":
    #        print("attr:", attrs)
            #print("Encountered a start tag:", tag)

    #def handle_endtag(self, tag):
    #    print("Encountered an end tag :", tag)

    #def handle_data(self, data):
    #    print("Encountered some data  :", data)


if __name__ == '__main__':
    main()

File: VoidDrops_auto/parse_prime.js
var request = require("request");
var cheerio = require("cheerio");

var pcBL = "https://forums.warframe.com/forum/3-pc-update-build-notes/";
var xbBL = "https://forums.warframe.com/forum/253-xb1-update-build-notes/";
var psBL = "https://forums.warframe.com/forum/152-ps4-update-build-notes/";

function getPrimeLink(html) {
    $ = cheerio.load(html);
    return $("a[href*='prime-gear-drop-locations']").first().attr('href');
};

request(pcBL, function(error,response,body) {
    if (!error) {
        var primeLink = getPrimeLink(body);
        console.log(primeLink);
        request(primeLink, function(error,response,body){
            console.log(body);
        })
    } else {
        console.log(error);
    }
});